src/CoreTests/DefaultRetryPolicyTests.cs

```diff
@@ -1,134 +0,0 @@
-using System;
-using System.Threading;
-using System.Threading.Tasks;
-using Marten;
-using Marten.Testing.Harness;
-using Npgsql;
-using Shouldly;
-using Weasel.Postgresql;
-using Xunit;
-
-namespace CoreTests;
-
-public class DefaultRetryPolicyTests
```

Comment:
**SUGGESTION:** I think that we should still have tests ensuring that the Polly default policy is applied as expected.